src/pyvesync/vesync.py

```diff
@@ -455,6 +528,17 @@ async def async_call_api(
             LibraryLogger.log_api_exception(logger, exception=e, request_body=req_dict)
             raise
 
+    def _api_base_url_for_current_region(self) -> str:
+        """Retrieve the API base url for the current region.
```

Comment:
Curious why you chose this way, instead of the US, CA, MX, JP being the .com and all else being the .eu url. The logic in the version I decompiled was call a configuration endpoint with the list of `.com` country codes (US CA MX JP) and if country code not in that list then eu.

Reply:
You can commit to my branch of you want, otherwise I'll do it on Monday since I'll be busy the rest of the weekend.